lib/plugins/interactiveCli/index.js

```diff
@@ -1,24 +1,19 @@
 'use strict';
 
-const _ = require('lodash');
 const inquirer = require('./inquirer');
 const initializeService = require('./initializeService');
 const setupAws = require('./setupAws');
 
 module.exports = class InteractiveCli {
   constructor(serverless) {
-    if (!process.stdin.isTTY) return;
+    this.serverless = serverless;
 
-    const { processedInput } = serverless;
-    if (processedInput.commands.length) return;
-    if (!_.isEmpty(processedInput.options)) return;
+    serverless.allowedInteractiveCliOptions = new Set();
 
     // Expose customized inquirer, and setupAws configuration for other plugins
     // setupAws is further customized by dashboard plugin
-    serverless.interactiveCli = { inquirer, awsSetupConfiguration: setupAws };
+    this.serverless.interactiveCli = { inquirer, awsSetupConfiguration: setupAws };
```

Comment:
This also should be set in async init, as existence of this object is decisive, whether we're in interactive CLI more or not.

Reply:
oh. duh.. 🤦‍♂